packages/gestalt/src/Popover/InternalPopover.js

```diff
@@ -0,0 +1,157 @@
+  /**
+   * Callback fired when Popover requests to be closed. Must be used to control Popover’s on/off display state.
+   */
+  onDismiss: () => void,
+  /**
+   * Properly positions Popover relative to its anchor element. Set to false when used within [Layer](https://gestalt.pinterest.systems/web/layer). See the [with Layer](https://gestalt.pinterest.systems/web/popover#With-layer) variant to learn more.
+   */
+  positionRelativeToAnchor?: boolean,
+  /**
+   * The underlying ARIA role for Popover. See the [accessibility](https://gestalt.pinterest.systems/web/popover#ARIA-attributes) section for more info.
+   */
+  role?: Role,
+  /**
+   * Puts the focus on Popover when it’s triggered. See [accessibility](https://gestalt.pinterest.systems/web/popover#Accessibility) to learn more.
+   */
+  shouldFocus?: boolean,
+  /**
+   * This field is deprecated and will be removed soon. Please do not use. See [PopoverEducational](https://gestalt.pinterest.systems/web/popovereducational).
+   */
+  showCaret?: boolean,
+  /**
+   * Shows a dismiss button on Popover. See the [dismiss button](https://gestalt.pinterest.systems/web/popover#Dismiss-button) variant to learn more.
+   */
+  showDismissButton?: boolean,
+  /**
+   * The maximum width of Popover. See the [size](https://gestalt.pinterest.systems/web/popover#Size) variant to learn more.
+   */
+  size?: Size,
+  // This property can be set when `ScrollBoundaryContainer` is set to `overflow="visible"` but therefore limits the height of the Popover-based component. Some cases require
+  __dangerouslySetMaxHeight?: '30vh',
+|};
+
+/**
+ * [Popover](https://gestalt.pinterest.systems/web/popover) is a floating view that contains a task related to the content on screen. It can be triggered when the user clicks or focuses on an element, typically [Button](https://gestalt.pinterest.systems/web/button) or [IconButton](https://gestalt.pinterest.systems/web/iconbutton). It can also be triggered automatically, as in the case of user education. Popover is non-modal and can be dismissed by interacting with another part of the screen or an item within Popover.
+ *
+ * Popover is most appropriate for desktop screens and can contain a variety of elements, such as [Button](https://gestalt.pinterest.systems/web/button) and [Images](https://gestalt.pinterest.systems/web/image). Popover is also the container used to construct more complex elements like [Dropdown](https://gestalt.pinterest.systems/web/dropdown) and the board picker, pictured below, which allow people to choose the board to save a Pin to.
+ *
+ * ![Popover light mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/Popover.spec.mjs-snapshots/Popover-chromium-darwin.png)
+ * ![Popover dark mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/Popover-dark.spec.mjs-snapshots/Popover-dark-chromium-darwin.png)
+ *
+ */
+export default function Popover({
+  accessibilityLabel = 'Popover',
```

Comment:
these is pretty nit.

I think these defaults props can be removed as we are setting them on the external Popover. 

If we do that let's make sure we don't lose them on PopoverEducational, unless we are overwriting them on PopoverEducational